README.md

```diff
@@ -14,5 +14,6 @@ one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,
 ## My Response
 Two graphs that meet the requirements do not have to be fully connected.
 Let $A$ and $B$ both be graphs, and let them each contain only one node.  This means that they have no edges, and are completely disconnected.
+Then let $f$ be a function that maps $A$ to $B$.
```

Comment:
You need to be more specific. From what you say here, $f$ could map all nodes in $A$ to one node in $B$, which wouldn't work for showing isomorphism. How is this function set up, and how does it map from one graph to the other?